include/base/QnTools/DataContainer.hpp

```diff
@@ -407,7 +408,11 @@ class DataContainer : public TObject {
             ++iprojbin;
           }
         }
-        projection.At(projindices) = lambda(projection.At(projindices), bin);
+        if (initialized_bins.insert(projindices).second) {
```

Comment:
So this check handles the first time initialization of a bin so if we project bins a1, a2, a3 on b0. 
1. b0 = a1; instead of b0 = b0 + a1;
2. b0 = b0 + a2;
3. b0 = b0 + a3;

Reply:
So we don't need to have the checks in the Merge function if we have it in the container. Previously I had a check in the Merge function, which would prefer the initialization of the right operand to circumvent these not initialized bins. But I think here is the better place for it.